commands/team.go

```diff
@@ -335,21 +336,23 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	if public {
 		privacy = model.TeamOpen
 	}
-
+	var Error error
 	teams := getTeamsFromTeamArgs(c, args)
 	for i, team := range teams {
 		if team == nil {
+			Error = multierror.Append(Error, fmt.Errorf("Unable to find team '" + args[i] + "'"))
```

Comment:
```suggestion
			Error = multierror.Append(Error, fmt.Errorf("Unable to find team %q", args[i]))
```
You can use a placeholder to put the argument into the error. Most function in `fmt` that end in `f` can do this (`Printf`, `Errorf`, `Sprintf`, etc).